controller/Lists.php

```diff
@@ -168,27 +170,39 @@ public function reloadRemoteList(ValueCollectionService $valueCollectionService,
         $this->assertIsXmlHttpRequest();
 
         $saved = false;
+        $message = __('Attempt for reloading of remote list was not successful');
+
         $uri = $_POST['uri'] ?? null;
 
         if ($uri !== null) {
-            $decodedUri = tao_helpers_Uri::decode($uri);
-
-            $this->sync(
-                $valueCollectionService,
-                $remoteSource,
-                $this->getListService()->getList($decodedUri)
-            );
+            try {
+                $this->sync(
+                    $valueCollectionService,
+                    $remoteSource,
+                    $this->getListService()->getList(tao_helpers_Uri::decode($uri)),
+                    true
+                );
 
-            $saved = true;
+                $saved = true;
+                $message = __('Remote list was successfully reloaded');
+            } catch (Throwable $exception) {
+                if ($exception instanceof ValueConflictException) {
+                    $message = $exception->getUserMessage();
+                }
+            }
         }
 
-        $this->returnJson(['saved' => $saved]);
+        $this->returnJson([
+            'saved' => $saved,
+            'message' => $message,
+        ]);
     }
 
     public function sync(
         ValueCollectionService $valueCollectionService,
         RemoteSource $remoteSource,
-        core_kernel_classes_Class $collectionClass
+        core_kernel_classes_Class $collectionClass,
+        bool $isReloading = false
```

Comment:
Generally bool parameters are a code smell for public interfaces. Is there a possibility to make this method private? It seems only used here.

Reply:
I think, yes, we can try to make this one private 🙂
I will check code one more time and will do this if possible